charts/keycloak/templates/headless-service.yaml

```diff
@@ -1,4 +1,5 @@
 {{- $highAvailability := gt (int .Values.keycloak.replicas) 1 -}}
+{{- $service := .Values.keycloak.service -}}
```

Comment:
Why is this needed to add labels?

Reply:
I guess we don't need it anymore!  I have updated the `verify_k8s_labels.py` verification script so it looks into labels on pods, not labels on services.